MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -37,10 +37,19 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
             "one is expected.",
             node_ids.size(), _point[0], _point[1], _point[2]);
 
+    auto& mesh_nodes =
+        const_cast<std::vector<MeshLib::Node*>&>(_mesh.getNodes());
+
     std::array<MeshLib::Node*, 1> const nodes = {{
         const_cast<MeshLib::Node*>(_mesh.getNode(node_ids[0]))}};
 
     _boundary_elements.push_back(new MeshLib::Point{nodes, node_ids[0]});
+    for (auto const* bulk_element : _mesh.getNode(node_ids[0])->getElements())
+    {
+        _bulk_ids.emplace_back(std::make_pair(
+            bulk_element->getID(),
+            bulk_element->identifyFace(mesh_nodes.data()+node_ids[0])));
```

Comment:
(Not thir PR) That `identifyFace` does not have a const argument looks like a mistake.